Atari-Game-Player/ReadMe.md

```diff
@@ -0,0 +1,49 @@
+# Atari - Solving Games with AI 🤖
+
+The purpose of this repository is to demonstrate how ZenML enables you to build complex applications in an easy way and Use ZenML for various classes of tasks.

```

Comment:
Maybe this is too broad a problem. Start with the fact that it is about reinforcement learning particularly

Reply:
Ok Thanks